.github/workflows/docker/Dockerfile-compilation-testing-alinux2

```diff
@@ -27,14 +29,19 @@ ENV PATH         $PATH:$GOROOT/bin:$GOPATH/bin
 ENV GOPROXY      "https://mirrors.aliyun.com/goproxy,direct"
 ENV GO111MODULE  on
 
-# install SGX
-RUN wget https://mirrors.openanolis.org/inclavare-containers/alinux2/sgx_linux_x64_sdk_2.13.100.4.bin && \
-    chmod +x sgx_linux_x64_sdk_2.13.100.4.bin && echo -e 'no\n/opt/intel\n' | ./sgx_linux_x64_sdk_2.13.100.4.bin && \
-    rm -rf sgx_linux_x64_sdk_2.13.100.4.bin
+# Configure Alibaba Cloud TEE SDK yum repo
+RUN yum-config-manager --add-repo \
+	https://enclave-cn-beijing.oss-cn-beijing.aliyuncs.com/repo/alinux/enclave-expr.repo
+
+# Install SGX Runtime
+RUN yum install --nogpgcheck -y libsgx-ae-le libsgx-ae-pce libsgx-ae-qe3 libsgx-ae-qve \
+    libsgx-aesm-ecdsa-plugin libsgx-aesm-launch-plugin libsgx-aesm-pce-plugin libsgx-aesm-quote-ex-plugin \
+    libsgx-dcap-default-qpl libsgx-dcap-ql libsgx-dcap-quote-verify \
+    libsgx-enclave-common libsgx-launch libsgx-pce-logic libsgx-qe3-logic libsgx-quote-ex \
+    libsgx-ra-network libsgx-ra-uefi libsgx-uae-service libsgx-urts sgx-ra-service \
+    sgx-aesm-service
```

Comment:
Do we need to install all these RPMs or just install `libsgx-dcap-quote-verify-devel libsgx-dcap-ql-devel libsgx-uae-service`